aura-borealis-flask-app/live_data2.py

```diff
@@ -0,0 +1,397 @@
+"""
```

Comment:
What is this live_data2.py file? How is it different from live_data.py? Can you please put in a comment at the top of both files that explains the difference.

Reply:
it was a backup, deleting, thanks!